src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.sln

```diff
@@ -17,7 +17,22 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevAdventCalendarCompetitio
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevAdventCalendarCompetition.TestResultService", "DevAdventCalendarCompetition.TestResultService\DevAdventCalendarCompetition.TestResultService.csproj", "{61350750-F9BB-4569-9BC4-0EF7CBC23C8C}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DevAdventCalendarCompetition.TestResultService.Tests", "DevAdventCalendarCompetition.TestResultService.Tests\DevAdventCalendarCompetition.TestResultService.Tests.csproj", "{8BCF955A-A084-42D6-9D5A-A3E8FC10FFB2}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevAdventCalendarCompetition.TestResultService.Tests", "DevAdventCalendarCompetition.TestResultService.Tests\DevAdventCalendarCompetition.TestResultService.Tests.csproj", "{8BCF955A-A084-42D6-9D5A-A3E8FC10FFB2}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{2DCD7FDB-32E7-4C8E-AFE4-E6C2C0195503}"
+	ProjectSection(SolutionItems) = preProject
+		.editorconfig = .editorconfig
+		Analysers.ruleset = Analysers.ruleset
```

Comment:
here I just put it into sln file to make it easier to get and edit, to have it in one place - solution. it doesn't mean that it will take part in any compilation.